--- a/glslang/Include/Common.h
+++ b/glslang/Include/Common.h
@@ -47,2 +47,3 @@
 #include <cstdio>
+#include <cstdint>
 #include <cstdlib>

--- a/glslang/Include/ConstantUnion.h
+++ b/glslang/Include/ConstantUnion.h
@@ -39,6 +39,6 @@
 #define _CONSTANT_UNION_INCLUDED_
 
-#include "../Include/Common.h"
-#include "../Include/BaseTypes.h"
+#include "Common.h"
+#include "BaseTypes.h"
 
 namespace glslang {

--- a/glslang/Include/InfoSink.h
+++ b/glslang/Include/InfoSink.h
@@ -36,4 +36,4 @@
 #define _INFOSINK_INCLUDED_
 
-#include "../Include/Common.h"
+#include "Common.h"
 #include <cmath>

--- a/glslang/Include/intermediate.h
+++ b/glslang/Include/intermediate.h
@@ -54,7 +54,7 @@
 #endif
 
-#include "../Include/Common.h"
-#include "../Include/Types.h"
-#include "../Include/ConstantUnion.h"
+#include "Common.h"
+#include "Types.h"
+#include "ConstantUnion.h"
 
 namespace glslang {

--- a/glslang/Include/Types.h
+++ b/glslang/Include/Types.h
@@ -41,5 +41,5 @@
 #define _TYPES_INCLUDED
 
-#include "../Include/Common.h"
-#include "../Include/BaseTypes.h"
+#include "Common.h"
+#include "BaseTypes.h"
 #include "../Public/ShaderLang.h"

--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -125,2 +125,3 @@
     ],
+    includes=["glslang","."],
     copts = COMMON_COPTS,

--- a/glslang/MachineIndependent/preprocessor/PpContext.h
+++ b/glslang/MachineIndependent/preprocessor/PpContext.h
@@ -84,4 +84,4 @@
 #include <sstream>
 
-#include "../ParseHelper.h"
+#include <MachineIndependent/ParseHelper.h>
 #include "PpTokens.h"

--- a/glslang/Include/Types.h
+++ b/glslang/Include/Types.h
@@ -44,3 +44,3 @@
 #include "BaseTypes.h"
-#include "../Public/ShaderLang.h"
+#include <Public/ShaderLang.h>
 #include "arrays.h"

--- a/glslang/Public/ShaderLang.h
+++ b/glslang/Public/ShaderLang.h
@@ -38,6 +38,6 @@
 #define _COMPILER_INTERFACE_INCLUDED_
 
-#include "../Include/ResourceLimits.h"
-#include "../MachineIndependent/Versions.h"
+#include <Include/ResourceLimits.h>
+#include <MachineIndependent/Versions.h>
 
 #include <cstring>

--- a/glslang/MachineIndependent/Initialize.h
+++ b/glslang/MachineIndependent/Initialize.h
@@ -38,6 +38,6 @@
 #define _INITIALIZE_INCLUDED_
 
-#include "../Include/ResourceLimits.h"
-#include "../Include/Common.h"
-#include "../Include/ShHandle.h"
+#include <Include/ResourceLimits.h>
+#include <Include/Common.h>
+#include <Include/ShHandle.h>
 #include "SymbolTable.h"

--- a/glslang/MachineIndependent/SymbolTable.h
+++ b/glslang/MachineIndependent/SymbolTable.h
@@ -66,6 +66,6 @@
 //
 
-#include "../Include/Common.h"
-#include "../Include/intermediate.h"
-#include "../Include/InfoSink.h"
+#include <Include/Common.h>
+#include <Include/intermediate.h>
+#include <Include/InfoSink.h>
 
 namespace glslang {

--- a/OGLCompilersDLL/InitializeDll.h
+++ b/OGLCompilersDLL/InitializeDll.h
@@ -35,5 +35,5 @@
 #define __INITIALIZEDLL_H
 
-#include "../glslang/OSDependent/osinclude.h"
+#include <OSDependent/osinclude.h>
 
 namespace glslang {

--- a/glslang/Include/ShHandle.h
+++ b/glslang/Include/ShHandle.h
@@ -45,4 +45,4 @@
 #define SH_EXPORTING
-#include "../Public/ShaderLang.h"
-#include "../MachineIndependent/Versions.h"
+#include <Public/ShaderLang.h>
+#include <MachineIndependent/Versions.h>
 #include "InfoSink.h"

--- a/glslang/MachineIndependent/attribute.h
+++ b/glslang/MachineIndependent/attribute.h
@@ -38,6 +38,6 @@
 #define _ATTRIBUTE_INCLUDED_
 
-#include "../Include/Common.h"
-#include "../Include/ConstantUnion.h"
+#include <Include/Common.h>
+#include <Include/ConstantUnion.h>
 
 namespace glslang {

--- a/glslang/MachineIndependent/ParseHelper.h
+++ b/glslang/MachineIndependent/ParseHelper.h
@@ -51,3 +51,3 @@
 #include "parseVersions.h"
-#include "../Include/ShHandle.h"
+#include <Include/ShHandle.h>
 #include "SymbolTable.h"

--- a/glslang/MachineIndependent/localintermediate.h
+++ b/glslang/MachineIndependent/localintermediate.h
@@ -40,5 +40,5 @@
 #define _LOCAL_INTERMEDIATE_INCLUDED_
 
-#include "../Include/intermediate.h"
-#include "../Public/ShaderLang.h"
+#include <Include/intermediate.h>
+#include <Public/ShaderLang.h>
 #include "Versions.h"

--- a/glslang/MachineIndependent/parseVersions.h
+++ b/glslang/MachineIndependent/parseVersions.h
@@ -40,5 +40,5 @@
 #define _PARSE_VERSIONS_INCLUDED_
 
-#include "../Public/ShaderLang.h"
-#include "../Include/InfoSink.h"
+#include <Public/ShaderLang.h>
+#include <Include/InfoSink.h>
 #include "Scan.h"

--- a/glslang/HLSL/hlslAttributes.h
+++ b/glslang/HLSL/hlslAttributes.h
@@ -40,5 +40,5 @@
 #include <functional>
 
-#include "../MachineIndependent/attribute.h"
-#include "../MachineIndependent/SymbolTable.h"
+#include <MachineIndependent/attribute.h>
+#include <MachineIndependent/SymbolTable.h>
 #include "hlslScanContext.h"

--- a/glslang/HLSL/hlslParseables.h
+++ b/glslang/HLSL/hlslParseables.h
@@ -37,5 +37,5 @@
 #define _HLSLPARSEABLES_INCLUDED_
 
-#include "../MachineIndependent/Initialize.h"
+#include <MachineIndependent/Initialize.h>
 
 namespace glslang {

--- a/glslang/HLSL/hlslParseHelper.h
+++ b/glslang/HLSL/hlslParseHelper.h
@@ -37,7 +37,7 @@
 #define HLSL_PARSE_INCLUDED_
 
-#include "../MachineIndependent/parseVersions.h"
-#include "../MachineIndependent/ParseHelper.h"
-#include "../MachineIndependent/attribute.h"
+#include <MachineIndependent/parseVersions.h>
+#include <MachineIndependent/ParseHelper.h>
+#include <MachineIndependent/attribute.h>
 
 #include <array>

--- a/glslang/HLSL/hlslScanContext.h
+++ b/glslang/HLSL/hlslScanContext.h
@@ -42,4 +42,4 @@
 #define HLSLSCANCONTEXT_H_
 
-#include "../MachineIndependent/ParseHelper.h"
+#include <MachineIndependent/ParseHelper.h>
 #include "hlslTokens.h"

--- a/glslang/MachineIndependent/LiveTraverser.h
+++ b/glslang/MachineIndependent/LiveTraverser.h
@@ -36,4 +36,4 @@
 #pragma once
 
-#include "../Include/Common.h"
+#include <Include/Common.h>
 #include "reflection.h"

--- a/glslang/MachineIndependent/propagateNoContraction.h
+++ b/glslang/MachineIndependent/propagateNoContraction.h
@@ -40,5 +40,5 @@
 #pragma once
 
-#include "../Include/intermediate.h"
+#include <Include/intermediate.h>
 
 namespace glslang {

--- a/glslang/MachineIndependent/reflection.h
+++ b/glslang/MachineIndependent/reflection.h
@@ -39,6 +39,6 @@
 #define _REFLECTION_INCLUDED
 
-#include "../Public/ShaderLang.h"
-#include "../Include/Types.h"
+#include <Public/ShaderLang.h>
+#include <Include/Types.h>
 
 #include <list>

--- a/glslang/MachineIndependent/ShaderLang.cpp
+++ b/glslang/MachineIndependent/ShaderLang.cpp
@@ -51,16 +51,16 @@
 #include "ScanContext.h"
 
 #ifdef ENABLE_HLSL
-#include "../HLSL/hlslParseHelper.h"
-#include "../HLSL/hlslParseables.h"
-#include "../HLSL/hlslScanContext.h"
+#include <HLSL/hlslParseHelper.h>
+#include <HLSL/hlslParseables.h>
+#include <HLSL/hlslScanContext.h>
 #endif
 
-#include "../Include/ShHandle.h"
-#include "../../OGLCompilersDLL/InitializeDll.h"
+#include <Include/ShHandle.h>
+#include <OGLCompilersDLL/InitializeDll.h>
 
 #include "preprocessor/PpContext.h"
 
 #define SH_EXPORTING
-#include "../Public/ShaderLang.h"
+#include <Public/ShaderLang.h>
 #include "reflection.h"