diff --git a/src/traced/probes/sys_stats/sys_stats_data_source.h
--- a/src/traced/probes/sys_stats/sys_stats_data_source.h
+++ b/src/traced/probes/sys_stats/sys_stats_data_source.h
@@ -23,2 +23,3 @@
 #include <memory>
+#include <optional>
 #include <string>

diff --git a/src/trace_processor/util/regex.h
--- a/src/trace_processor/util/regex.h
+++ b/src/trace_processor/util/regex.h
@@ -23,3 +23,3 @@
   }
-  Regex(Regex&) = delete;
+  Regex(const Regex&) = delete;
   Regex(Regex&& other) {

diff --git a/src/trace_processor/perfetto_sql/engine/created_function.cc
--- a/src/trace_processor/perfetto_sql/engine/created_function.cc
+++ b/src/trace_processor/perfetto_sql/engine/created_function.cc
@@ -19,2 +19,3 @@
 #include <cstddef>
+ using std::nullptr_t;
 #include <queue>
    
--- a/include/perfetto/ext/base/sys_types.h
+++ b/include/perfetto/ext/base/sys_types.h
@@ -28,6 +28,6 @@
 #if PERFETTO_BUILDFLAG(PERFETTO_OS_WIN)
 
+using uid_t = unsigned int;
 #if !PERFETTO_BUILDFLAG(PERFETTO_COMPILER_GCC)
 // MinGW has these. clang-cl and MSVC, which use just the Windows SDK, don't.
-using uid_t = int;
 using pid_t = int;
